### ISSUE TITLE
 Is NessieViewCatalog available?

### ISSUE THREAD

**Issue by melin** [projectnessie/nessie#3658]
Is NessieViewCatalog available? Spark ViewCatalog will be available for more than a year.
@nastra

**Comment by nastra** [comment]
@melin there is just an experimental implementation of `NessieViewCatalog` available at this moment, that understands how to deal with Views and map those to Nessie. This will eventually be moved into Iceberg itself once the View Spec + the View implementation itself is merged into Iceberg.
Spark ViewCatalog is a completely separate thing and as you mentioned, it's not available at this time.
